### DISCUSSION TITLE
【咨询】关于宇宙的一些探讨

### DISCUSSION THREAD

**Discussion by Aisyle-001** [share121/inter-knot#1409] [其他]
**你好，刚从厕所出来。
刚刚并非停止了思考，而是有点失落，自认为无法解惑。
但是今天整理下思路，会不会是这样。
宇宙最初是0，在某个点上发生了崩坏和撕裂，并且暴涨开去，让整个宇宙所及的范围内都发生了撕裂。
从0撕开了正数物质和负数物质。
正数产生了我们的世界，有引力，聚成团，发光，发热。
负数产生了和我们在一起的另一个世界，只是那个世界的引力是斥力，它推动我们的物质分散开去，也就是宇宙的膨胀。
我们的指缝间，我和月球间，我们和太阳间充满了这些物质，只是太过于稀薄，且是斥力所以不发生关系和湮灭。
即使有一丁点的湮灭，也是我们不可观测的规模。
更可怕的是，他们充满了银河系，充满了整个超星系团，和整个宇宙，只是太稀薄，所以基本无法观测。
但是，体量巨大，和我们可见宇宙刚好相等，一样多。
黑洞在让正物质坠入其中，却让负物质远离。
空间的扭曲和反扭曲，这种原理是否可以被我们的星际航行所用。
宇宙正物质衰变成夸克汤后，是否正好和负宇宙完全接触，来一次彻底的湮灭，归零？
抱歉，又要去一下厕所，等下聊。**

**Comment by Dave-12138** [comment]
什么祭礼枪

**Comment by gaojiaobei** [comment]
神器————祭礼枪！

**Comment by qiuxi903** [comment]
贤者模式，启动！
